assets/js/script.js

```diff
@@ -150,48 +168,65 @@ function render() {
   $('.js-tab').html('')
   $('.js-preview').html('')
   $('.js-tab').removeClass('active')
+  $('.js-tab').parent().removeClass('active')
   $('.js-swatch').removeClass('active')
   $('.js-loading').hide()
 
   // render bag color
-  $('.js-bag').css({backgroundImage: "url("+(window.baseUrl || '')+"assets/img/bags/danny-"+state.bag+".png)"})
 
-  // render letters in bag and preview
+  $('.js-bag').css({ backgroundImage: "url(" + (window.baseUrl || '') + "assets/img/bags/danny-" + state.bag + ".png)" })
+
+  $('.js-tab').each(function() {
+      if ($(this).data('index') != -1) {
```

Comment:
super minor trick you can use for readability, if you have a short escape condition and a long `else` block:

```js
if ($(this).data('index') == -1) { return $(this).html('All') }

var ind = $(this).data('index')
var src = `${window.baseUrl || ''}assets/img/letters/` + state.letters[ind].toUpperCase() + `-` + state.materials[ind] + `.png`;
$(this).css({ background: "url(" + (window.baseUrl || '') + src + ") no-repeat", backgroundSize: "contain", backgroundPosition: "center" })
```